.github/scripts/github_metrics.py

```diff
@@ -0,0 +1,91 @@
+"""
+Submit metrics returns "Payload accepted" response
+"""
+
+from datetime import datetime
+from datetime import timedelta
+from datadog_api_client.v1 import ApiClient, Configuration
+from datadog_api_client.v1.api.metrics_api import MetricsApi
+from datadog_api_client.v1.model.metrics_payload import MetricsPayload
+from datadog_api_client.v1.model.point import Point
+from datadog_api_client.v1.model.series import Series
+import requests
+
+
+github_url="https://api.github.com/repos/twilio/twilio-cli/releases"
+response = requests.get(github_url)
+output=response.json()
+rpm_count = deb_count = exe_count = pkg_count = 0
+
+for release in output:
```

Comment:
Does this mean that the output json will have all releases in response and this loop would go through all of them, each time?

Reply:
It would have the data for the last 30 releases

Reply:
Okay. This is the data we will use to plot the graph then?

Reply:
Yes